works.xml

```diff
@@ -18380,8 +18380,8 @@ Titre collectif pour deux séries de commentaires théologiques de passages diff
             </bibl>
             <bibl xml:id="work_5607">
                <author key="person_2485707">Cranmer, Thomas, 1489–1556</author>
-               <title type="uniform">Cranmer, Thomas, 1489–1556: Certayne sermons appoynted by the Quenes Maiestie to be declared and read, by all persones, vycars, and curates, euery Sonday and holy daye, in theyr Churches [English]</title>
-               <textLang mainLang="en"/>
+               <title type="uniform">Cranmer, Thomas, 1489–1556: Certayne sermons appoynted by the Quenes Maiestie to be declared and read, by all persones, vycars, and curates, euery Sonday and holy daye, in theyr Churches [Middle English]</title>
+               <textLang mainLang="enm"/>
```

Comment:
This is the sort of marginal case that makes these distinctions tricky. Is this still Middle English? not sure...